larray_editor/editor.py

```diff
@@ -665,6 +674,33 @@ def open_tutorial(self):
     def open_api_documentation(self):
         QDesktopServices.openUrl(QUrl("http://larray.readthedocs.io/en/stable/api.html"))
 
+    def about(self):
+        """About Editor"""
+        versions = get_versions()
+        message = \
+            """<b>LArray Editor {editor}</b>
+            <br>The Graphical User Interface for LArray. This project aims to provide an intuitive tool to 
```

Comment:
About dialogs usually do not describe what the program is, so I would keep only "The Graphical User Interface for LArray".